PAINS filters update fails when old Python is installed

Here's what happens on Centos5 (python v2.4):
```
[vagrant@localhost centos5_64]$ BOOST_ROOT=/usr/local JAVA_HOME=/opt/jdk1.7.0_79 cmake -DBoost_USE_STATIC_LIBS=ON -DRDK_BUILD_AVALON_SUPPORT=ON -DRDK_BUILD_INCHI_SUPPORT=ON -DRDK_BUILD_SWIG_WRAPPERS=ON -DRDK_BUILD_PYTHON_WRAPPERS=OFF ../../..
-- The C compiler identification is GNU 4.1.2
-- The CXX compiler identification is GNU 4.1.2
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works

   SNIP

-- Found PythonInterp: /usr/bin/python (found version "2.4.3") 
== Updating Filters.cpp from pains file
  File "/home/vagrant/RDKit_git/Code/GraphMol/FilterCatalog/update_pains.py", line 2
    from __future__ import print_function
SyntaxError: future feature print_function is not defined
CMake Error at Code/GraphMol/FilterCatalog/CMakeLists.txt:16 (message):
  Failed updating Filters.cpp from wehi_pains.csv


-- Configuring incomplete, errors occurred!
See also "/home/vagrant/RDKit_git/builds/centos5/centos5_64/CMakeFiles/CMakeOutput.log".
See also "/home/vagrant/RDKit_git/builds/centos5/centos5_64/CMakeFiles/CMakeError.log".
```

The solution is to include a check for Python version as well as existence in the CMakeLists.txt file.

--- a/Code/GraphMol/FilterCatalog/CMakeLists.txt
+++ b/Code/GraphMol/FilterCatalog/CMakeLists.txt
@@ -5,7 +5,7 @@ else()
 endif()
 
 find_package(PythonInterp)
-if (PYTHONINTERP_FOUND)
+if (PYTHONINTERP_FOUND AND ${PYTHON_VERSION_STRING} VERSION_GREATER "2.6" )
       message("== Updating Filters.cpp from pains file")
       execute_process(
         COMMAND
@@ -15,7 +15,7 @@ if (PYTHONINTERP_FOUND)
       if(NOT rc EQUAL 0)
         message(FATAL_ERROR "Failed updating Filters.cpp from wehi_pains.csv")
       endif(NOT rc EQUAL 0)
-endif(PYTHONINTERP_FOUND)
+endif()
 
 rdkit_library(FilterCatalog
               Filters.cpp

